docs/user/filesystem.md

```diff
@@ -974,6 +974,13 @@ libpil4dfs intercepting summary for ops on DFS:
 [op_sum ]  5003
 ```
 
+### Change dir cache timeout with env
+
+Directory caching is employed for better performance. The default timeout is 10 seconds. User can change it if necessary. The unit is second and an integer should be provided.
+```
+$ export D_IL_DCACHE_TIMEOUT=5
```

Comment:
This is the equivalent setting as dfuse-dentry-dir-time so it would be good to use a matching name.  Perhaps PIL4DFS_DENTRY_DIR_TIME given as this is a env var rather than a container attribute.